docs/core/tutorials/snippets/library-with-visual-studio/csharp/StringLibraryTest/StringLibraryTest.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
-    <PackageReference Include="MSTest.TestAdapter" Version="3.2.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="3.2.1" />
     <PackageReference Include="MSTest.TestFramework" Version="3.2.1" />
```

Comment:
I would suggest replacing these 3 lines with simply `<PackageReference Include="MSTest"  Version="3.2.1" />` that references the metapackage we offer and that makes sure all versions are aligned and correctly working together.

Reply:
Ah, I saw you mention lines 10-12. Done, see a6840a0.